tonic/src/transport/channel/service/tls.rs

```diff
@@ -33,7 +34,14 @@ impl TlsConnector {
         #[cfg(feature = "tls-native-roots")] with_native_roots: bool,
         #[cfg(feature = "tls-webpki-roots")] with_webpki_roots: bool,
     ) -> Result<Self, crate::BoxError> {
-        let builder = ClientConfig::builder();
+        let crypto_provider = match crypto::CryptoProvider::get_default() {
+            Some(default) => default.clone(),
```

Comment:
Can we just change `default` to `existing`?

Would be nice to avoid the clash with the `default` keyword given it's use in specialization.